Compiler/src/tfuncs.jl

```diff
@@ -450,6 +450,10 @@ end
                     return Const(true)
                 end
             end
+        # datatype_fieldcount is what `fieldcount` uses internally
+        # and returns nothing (!==0) for non-definite field counts rat
```

Comment:
Out of curiosity, what does "rat" stand for here?

Reply:
oh that's easy. It stands for I made a commit at 10pm and therefore forgot to finish my sentence mid-word